src/lib/operators/export_binary.hpp

```diff
@@ -176,9 +178,67 @@ class ExportBinary : public AbstractReadOnlyOperator {
   template <typename T>
   static void _write_segment(const RunLengthSegment<T>& run_length_segment, std::ofstream& ofstream);
 
-  // Chooses the right FixedSizeByteAlignedVector depending on the attribute_vector_width and exports it.
-  static void _export_attribute_vector(std::ofstream& ofstream, const CompressedVectorType type,
-                                       const BaseCompressedVector& attribute_vector);
+  /**
+   * FrameOfReference Segments are dumped with the following layout:
+   *
+   * Description            | Type                                  | Size in bytes
+   * -----------------------------------------------------------------------------------------
+   * Column Type            | ColumnType                            |   1
```

Comment:
Didn't see this in the code. Does this refer to `   export_value(ofstream, EncodingType::FrameOfReference); `?

Reply:
yes indeed. renamed it to Encoding Type.